terraform/aws/main.tf

```diff
@@ -28,9 +28,11 @@ provider "helm" {
 }
 
 module "vpc" {
-  source         = "../modules/aws/vpc"
-  env            = var.env
-  building_block = var.building_block
+  source             = "../modules/aws/vpc"
+  env                = var.env
+  building_block     = var.building_block
+  region             = var.region
+  availability_zones = var.availability_zones
```

Comment:
Why are we introducing the property to "vpc" module? How was it working in earlier days?

Reply:
Earlier we provided a default value for vpc module - https://github.com/Sanketika-Obsrv/obsrv-automation/blob/1b6b9db117bd2ea46e7777c4783fae5181f39de5/terraform/modules/aws/vpc/variables.tf#L35
So to default value for the variables it needs to at the directory where terraform is being executed.